clients/admin-ui/src/features/datastore-connections/ConnectionFilters.tsx

```diff
@@ -0,0 +1,179 @@
+  const filters = useSelector(selectPrivacyRequestFilters);
+  const token = useSelector(selectToken);
+  const dispatch = useDispatch();
+  const toast = useToast();
+  const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) =>
+    dispatch(setRequestId(event.target.value));
+  const handleStatusChange = (event: React.ChangeEvent<HTMLSelectElement>) =>
+    dispatch(setRequestStatus(event.target.value as PrivacyRequestStatus));
+  const handleFromChange = (event: React.ChangeEvent<HTMLInputElement>) =>
+    dispatch(setRequestFrom(event?.target.value));
+  const handleToChange = (event: React.ChangeEvent<HTMLInputElement>) =>
+    dispatch(setRequestTo(event?.target.value));
+  const handleClearAllFilters = () => dispatch(clearAllFilters());
+  const handleDownloadClick = async () => {
+    let message;
+    try {
+      await requestCSVDownload({ ...filters, token });
+    } catch (error) {
+      if (error instanceof Error) {
+        message = error.message;
+      } else {
+        message = "Unknown error occurred";
+      }
+    }
+    if (message) {
+      toast({
+        description: `${message}`,
+        duration: 5000,
+        status: "error",
+      });
+    }
+  };
+
+  return {
+    handleSearchChange,
+    handleStatusChange,
+    handleFromChange,
+    handleToChange,
+    handleClearAllFilters,
+    handleDownloadClick,
+    ...filters,
+  };
+};
+
+const DataStoreTypeOption: React.FC<{ status: ConnectionType }> = ({
+  status,
+}) => <option value={status}>{capitalize(status)}</option>;
+
+const SystemTypeOption: React.FC<{ status: SystemType }> = ({ status }) => (
+  <option value={status}>{capitalize(status)}</option>
+);
+
+const TestingStatusOption: React.FC<{ status: TestingStatus }> = ({
+  status,
+}) => <option value={status}>{capitalize(status)}</option>;
+
+const ConnectionFilters: React.FC = () => {
+  const { status, handleSearchChange, handleStatusChange, id } =
+    useConstantFilters();
+  return (
+    <Stack direction="row" spacing={4} mb={6}>
+      <InputGroup size="sm">
+        <InputLeftElement pointerEvents="none">
+          <SearchLineIcon color="gray.300" w="17px" h="17px" />
+        </InputLeftElement>
+        <Input
+          type="search"
+          minWidth={200}
+          placeholder="Search"
+          size="sm"
+          borderRadius="md"
+          value={id}
+          name="search"
+          onChange={handleSearchChange}
+        />
+      </InputGroup>
+      <Select
+        placeholder="Datastore Type"
+        size="sm"
+        minWidth="144px"
+        value={status || ""}
+        onChange={handleStatusChange}
+        borderRadius="md"
+      >
+        <DataStoreTypeOption status={ConnectionType.POSTGRES} />
+        <DataStoreTypeOption status={ConnectionType.MONGODB} />
+        <DataStoreTypeOption status={ConnectionType.MYSQL} />
+        <DataStoreTypeOption status={ConnectionType.HTTPS} />
+        <DataStoreTypeOption status={ConnectionType.SAAS} />
```

Comment:
This is similar to the issue we have with the SaaS connector logos. Selecting this option should really do the same thing as selecting SaaS from `System Type`. I'd suggest let's remove it here to avoid duplicating the functionality which will only serve to confuse our users and make the UI harder to maintain.

Reply:
I removed the `Saas` option from Datastore type dropdown 👍